File: NetFramework/WinForms/CS/ControlExplorer/RulesManager/Dialogs/BetweenForm.cs
using ControlExplorer.RulesManager.Presets;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace ControlExplorer.RulesManager.Dialogs
{
    public partial class BetweenForm : BaseConditionalFormattingForm
    {
        public BetweenForm()
        {
            InitializeComponent();
        }

        private void BetweenForm_Load(object sender, EventArgs e)
        {
            cmbStylePreset.SelectedIndex = 0;
        }

        [Browsable(false)]
        public Highlight SelectedStylePreset
        {
            get
            {
                var selectedStylePresetText = cmbStylePreset.Text;
                var stylePresets = Enum.GetValues(typeof(Highlight)).Cast<Highlight>().ToList();

                return stylePresets.First(stylePreset => stylePreset.ToString().ToLower() == selectedStylePresetText.ToLower().Replace(" ", ""));
            }
        }

        [Browsable(false)]
        public string MinimumValue
        {
            get
            {
                var textMinimumValue = tbMinimumValue?.Text;
                if (textMinimumValue != null)
                {
                    return textMinimumValue;
                }

                return string.Empty;
            }
        }

        [Browsable(false)]
        public string MaximumValue
        {
            get
            {
                var textMaximumValue = tbMaximumValue?.Text;
                if (textMaximumValue != null)
                {
                    return textMaximumValue;
                }

                return string.Empty;
            }
        }
    }
}

File: NetFramework/RulesManager/VB/ExcelLikeConditionalFormatting/Dialogs/BetweenForm.vb
Imports System.ComponentModel

Public Class BetweenForm
    Private Sub BetweenForm_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        cmbStylePreset.SelectedIndex = 0
    End Sub

    <Browsable(False)>
    Public ReadOnly Property SelectedStylePreset As Highlight
        Get
            Dim selectedStylePresetText = cmbStylePreset.Text
            Dim stylePresets = [Enum].GetValues(GetType(Highlight)).Cast(Of Highlight)().ToList()
            Return stylePresets.First(Function(stylePreset) stylePreset.ToString().ToLower() = selectedStylePresetText.ToLower().Replace(" ", ""))
        End Get
    End Property

    <Browsable(False)>
    Public ReadOnly Property MinimumValue As String
        Get
            Dim textMinimumValue = tbMinimumValue?.Text

            If textMinimumValue IsNot Nothing Then
                Return textMinimumValue
            End If

            Return String.Empty
        End Get
    End Property

    <Browsable(False)>
    Public ReadOnly Property MaximumValue As String
        Get
            Dim textMaximumValue = tbMaximumValue?.Text

            If textMaximumValue IsNot Nothing Then
                Return textMaximumValue
            End If

            Return String.Empty
        End Get
    End Property
End Class